client/packages/common/src/hooks/useSingleDeleteConfirmation/useSingleDeleteConfirmation.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  useTranslation,
+  useNotification,
+  useConfirmationModal,
+} from '@openmsupply-client/common';
+
+interface SingleDeleteConfirmationProps {
+  deleteAction: () => Promise<void>;
+  messages?: {
+    confirmTitle?: string;
+    confirmMessage?: string;
+    deleteSuccess?: string;
+    cantDelete?: string;
+    selectRows?: string;
+  };
+}
+
+export const useSingleDeleteConfirmation = ({
```

Comment:
Didn't need all the fields form useDeleteConfirmation, so have just created another component with the needed fields

Reply:
there's no need to create another hook. Simply pass in the row when calling `useDeleteConfirmation`
for example: adding this to the stocktake implementation

```
    selectedRows: [data],
```

is the only change required.
Can you update and remove the new hook please?

Reply:
[df344d0](https://github.com/openmsupply/open-msupply/pull/2218/commits/df344d0177cb7f795ae9dfe79b1cdad6ab2fa7a1)